### PR TITLE
adds necessary style sheet for sign on widget styling

### PR DESCRIPTION
**Pull request by jtmst**
## Description
Adds a style sheet that was missing in order for the sign on widget to style correctly.  There will be a seperate vets-website PR to address the crisis modal fix

Relates to #14802. (or closes?)
https://github.com/department-of-veterans-affairs/va.gov-cms/issues/14802
## Testing done
local visual testing
## Screenshots
image

## QA steps
Verify that sign on widget loaded from the main nav renders as expected

### Definition of Done

- [ ] Documentation has been updated, if applicable.
- [ ] Tests have been added if necessary.
- [ ] Automated tests have passed.
- [ ] Code Quality Tests have passed.
- [ ] Acceptance Criteria in related issue are met.
- [ ] Manual Code Review Approved.

### Is this PR blocked by another PR?

- [ ] `DO NOT MERGE`

### CODE REVIEW

**Comment by tjheffner** [comment]
I do see the sign in widget rendering on tugboat, but it feels a little off from va.gov still. Are other stylesheets missing that would effect this?

Note the difference in the margin on the right side of the open widget container
Screenshot 2023-08-23 at 10 24 43 AM
Screenshot 2023-08-23 at 10 24 48 AM

**Comment by jtmst** [comment]
> I do see the sign in widget rendering on tugboat, but it feels a little off from va.gov still. Are other stylesheets missing that would effect this?
> 
> Note the difference in the margin on the right side of the open widget container Screenshot 2023-08-23 at 10 24 43 AM Screenshot 2023-08-23 at 10 24 48 AM

I see the difference but it also looks like the styles are the same.  The css unit used for this is 'ch' which is derived from the font, and we are still having CORS issues preventing the correct fonts from loading in.  The margin issue should fix itself once the fonts are corrected

**Comment by jtmst** [comment]
looking at it again though, im seeing that the sign in button is mis-colored on hover, let me check that out before we merge

**Comment by tjheffner** [comment]
> I see the difference but it also looks like the styles are the same. The css unit used for this is 'ch' which is derived from the font, and we are still having CORS issues preventing the correct fonts from loading in. The margin issue should fix itself once the fonts are corrected

Copy that. I have noticed some issues related to `em` units being used as well (not in this PR, but with some font stuff in general). We'll have to fix the CORS stuff sooner than later (also not in this PR)

File: src/templates/globals/header/index.tsx [hunk 1]
```diff
@@ -2,7 +2,7 @@ import { TopNav } from './TopNav'
 
 export const Header = () => {
   return (
-    <header className="header" role="banner">
+    <header className="header merger" role="banner">
```
Comment (tjheffner) [inline src/templates/globals/header/index.tsx:5]: this fixes the header styles, but will you move this onto the `<body>` tag in `_document.tsx` instead? That's where I'm seeing it on va.gov, other non-header components may also need that class in their scope

Comment (jtmst) [reply src/templates/globals/header/index.tsx]: good catch, fixed.

**Review by tjheffner — APPROVED** [review]

**Pull request merged**